english/heffalump/heffalump.md

```diff
@@ -0,0 +1,3 @@
+A heffalump you say? What's one of those...
+
+Well it's a magical beast with [super powers](../super-powers/practice-flying.md). Not to be confused with a [scarab](../scarab/goddess.md), or in fact a [marshmallow](../marshmallow.md).
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.